requirements-win-cp27-cp34.txt

```diff
@@ -0,0 +1,5 @@
+--extra-index-url https://pypi.anaconda.org/carlkl/simple
+cmake
+cython
+mingwpy
+piksi_tools

```

Comment:
for what reason are piksi_tools required on python 2.7?  Is this just a convenient way to specify some other dependency that is in piksi_tools?

Reply:
This was for convenience for sanity checking. Removing.

Reply:
So pretty much what @benjaminaltieri said.